test/functional/combine_logs.py

```diff
@@ -19,22 +22,29 @@
 
 def main():
     """Main function. Parses args, reads the log files and renders them as text or html."""
-
-    parser = argparse.ArgumentParser(usage='%(prog)s [options] <test temporary directory>', description=__doc__)
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        'testdir', nargs='?', default='',
+        help=('temporary test directory to combine logs from. '
+              'Defaults to the most recent'))
     parser.add_argument('-c', '--color', dest='color', action='store_true', help='outputs the combined log with events colored by source (requires posix terminal colors. Use less -r for viewing)')
     parser.add_argument('--html', dest='html', action='store_true', help='outputs the combined log as html. Requires jinja2. pip install jinja2')
-    args, unknown_args = parser.parse_known_args()
+    args, _ = parser.parse_known_args()
 
     if args.html and args.color:
         print("Only one out of --color or --html should be specified")
         sys.exit(1)
 
-    # There should only be one unknown argument - the path of the temporary test directory
-    if len(unknown_args) != 1:
-        print("Unexpected arguments" + str(unknown_args))
+    testdir = args.testdir or find_latest_test_dir()
+
+    if not testdir:
+        print("No test directories found")
         sys.exit(1)
 
-    log_events = read_logs(unknown_args[0])
+    if not args.testdir:
+        print("Opening latest test directory: {}".format(testdir))
```

Comment:
Can you drop this print, or print to stderr.

(printing this is problematic when using the --html output option for example)